Spec/RestClient.swift

```diff
@@ -832,6 +832,24 @@ class RestClient: QuickSpec {
                 let object = try! NSJSONSerialization.JSONObjectWithData(transport.rawDataSent.first!, options: NSJSONReadingOptions(rawValue: 0))
                 expect(NSJSONSerialization.isValidJSONObject(object)).to(beTrue())
             }
+
+            // RSC7a
+            it("X-Ably-Version must be included in all REST requests") {
+                let options = AblyTests.commonAppSetup()
+                let client = ARTRest(options: options)
+                client.httpExecutor = testHTTPExecutor
+                waitUntil(timeout: testTimeout) { done in
+                    client.channels.get("test").publish(nil, data: "message") { error in
+                        expect(error).to(beNil())
+                        guard let headerAblyVersion = testHTTPExecutor.requests.first?.allHTTPHeaderFields?["X-Ably-Version"] else {
+                            fail("X-Ably-Version header not found"); done()
+                            return
+                        }
+                        expect(headerAblyVersion) == ARTDefault.version()
```

Comment:
There is already a G4 https://github.com/ably/ably-ios/blob/b3fdead2e828bf472bc51f3364e4118b8459b817/Spec/RestClient.swift#L32.
The current test is verifying if the client conforms with defaults.